src/neighbors.h

```diff
@@ -42,8 +42,10 @@ void neighbors_init(void);
 #if USE_HTCP
 void neighborsHtcpClear(StoreEntry *, HttpRequest *, const HttpRequestMethod &, htcp_clr_reason);
 #endif
-CachePeer *peerFindByName(const char *);
-CachePeer *peerFindByNameAndPort(const char *, unsigned short);
+
+/// cache_peer with a given name (or nil)
+CachePeer *findCachePeerByName(const char *);
```

Comment:
We do not have to rename peerFindByName() but I think it is a good idea because there were bugs in the official code where the old function was called with the wrong c-string parameter (a host name instead of cache_peer name). It is probably best to trigger build failures when this change is merged into branches where peerFindByName() is used incorrectly in contexts unknown/unchecked in this PR.